packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -77,7 +77,7 @@ function computeReplicaConfiguration(
   buildEnv: BuildEnv,
   intakeEndpoints: string[]
 ): ReplicaConfiguration | undefined {
-  if (buildEnv.buildMode !== BuildMode.STAGING || initConfiguration.replica === undefined) {
+  if (!initConfiguration.replica) {
```

Comment:
what about enabling replica only for canary mode?
it should not be used outside of DD applications, right?

Reply:
Right, 
I was wondering, if the goal is to prevent users from using it:
Not having the option officially documented is it not enough?
What about `enableExperimentalFeatures` then?

Reply:
I'd say that `enableExperimentalFeatures` is more for temporary experiment rather than long lasting code.
If we only need it on DD app, using a specific build mode has the advantage of not shipping related code to customers.

Reply:
I checked the consul config for eu, us3 and us5 and we use the stable v3 version on them. 
So, in order to allow the replica on these envs either:
- We remove the buildMode check to keep the stable version
- We add `buildEnv.buildMode !== BuildMode.CANARY` check and set the canary version also on these envs

My personal feeling, is that I prefer to keep the canary version only for US1 to scope our impact in case something went wrong but it's not a strong opinion. Wdyt?